torchgeo/samplers/single.py

```diff
@@ -257,7 +290,7 @@ class PreChippedGeoSampler(GeoSampler):
     and subclass :class:`~torchgeo.datasets.GeoDataset` but have already been
     pre-processed into :term:`chips <chip>`.
 
-    This sampler should not be used with :class:`~torchgeo.datasets.NonGeoDataset`.
+    This sampler should not be used with :class:`~torchgeo.datasets.VisionDataset`.
```

Comment:
This should be undone, we renamed the base class

Reply:
Yep! I'm currently on this. Will also make sure unit tests are adapted to this new (or old) solution